Casks/k/kodi.rb

```diff
@@ -10,15 +10,14 @@
   desc "Free and open-source media player"
   homepage "https://kodi.tv/"
 
-  # The regex below assumes that the release name will always be one word
-  # (e.g., Leia, Matrix, Nexus, Omega, etc.).
+  # The upstream website is protected by Cloudflare, which prevents
+  # us from fetching the download page. So we check the GitHub repository.
   livecheck do
-    url "https://kodi.tv/download/macos/"
-    regex(/href=.*?kodi[._-]v?(\d+(?:\.\d+)+[._-][^-]+?)[._-][^-]+?\.dmg/i)
+    url "https://github.com/xbmc/xbmc/releases"
```

Comment:
```suggestion
    url "https://github.com/xbmc/xbmc/"
```

The `GithubLatest` strategy will generate a new URL from the GitHub URL, so we don't need to include `/releases` in this context (I only include what's necessary).